packages/renderer-bootstrap4/renderers/textArea-field-renderer.tsx

```diff
@@ -26,10 +34,11 @@ const Bootstrap4TextAreaFieldRenderer = ({
       </Bootstrap4LabelRenderer>
       <textarea
         class={combineCssClasses('form-control', errors.length && 'is-invalid')}
+        ref={(r) => (textAreaRef = r)}
```

Comment:
This should be handled using a component, so you should convert this functional component into a class. See https://jsfiddle.net/developit/x8qpzjdf/ for an example. What you have set up now would cause the textAreaRef to be shared among renderers which is not what you want here.